test/templates/pages/floorplans/object.njk

```diff
@@ -5,9 +5,7 @@
 {% from "./../../table/component.njk" import table %}
 {% from "./../../button-group/component.njk" import button_group %}
 {% from "./../../toggle/component.njk" import toggle %}
-
```

Comment:
I commented out all of the forms code here. If anyone is using it/wants it to remain, I'll update the forms so they work here. Otherwise, we should delete the file.